configs/terminate_connect.yaml

```diff
@@ -56,9 +59,4 @@ static_resources:
             address:
               socket_address:
                 address: www.google.com
-                port_value: 443
-    transport_socket:
-      name: envoy.transport_sockets.tls
-      typed_config:
-        "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
-        sni: www.google.com
+                port_value: 80
```

Comment:
Why does the port value change?

Reply:
You're looking at an outdated version - the version submitted used 443 which should work :-)